java/src/jmri/jmrix/marklin/MarklinSystemConnectionMemo.java

```diff
@@ -19,6 +19,10 @@
  */
 public class MarklinSystemConnectionMemo extends jmri.jmrix.DefaultSystemConnectionMemo implements ConfiguringSystemConnectionMemo {
 
+    // Allow is-a relation between sub types, i.e. CdB and Marklin
+    protected MarklinSystemConnectionMemo(String prefix, String userName) {
```

Comment:
@bobjacobsen 
I suggest you change this method so that it only takes the userName, not the connection prefix. That will prevent `C` instead of `M`.

Reply:
That's part of the code that allows the user to specify their prefix.

After this gets merged, I'll do a little bit of work on this system to get it fully connected to the preferences Connection pane.  That'll use this constructor.